.github/workflows/macos.yml

```diff
@@ -3,14 +3,17 @@ name: macOS
 
 on:
   push:
-    branches: [master]
+    branches:
+      - "**"
   pull_request:
-    branches: [master]
+    branches:
+      - "master"
 
 jobs:
   build-and-test:
     name: Build and Test
     runs-on: macos-latest
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != '3rdparty/stout-grpc'
```

Comment:
Two things about this:
(1) Can we please add a comment explaining why this is here?
(2) Does this line make lines 9 and 10 (triggering on pull request to master) obsolete?

Reply:
1) Of course we can:)
2) Definitely not cause we wanna run this checks on external PRs:)

Reply:
Re 2: good point!